1648349888447	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1648349889537	Sync.SyncScheduler	DEBUG	Going to sleep, doing a quick sync.
1648349889538	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648349889538	Sync.Service	INFO	Starting sync at 2022-03-27 08:28:09 in browser session QYCP8427cDk4
1648349889538	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1648349889540	Sync.Status	INFO	Resetting Status.
1648349889540	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1648349889541	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1648351922515	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1648351922516	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1648351922518	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1648351922522	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648351922563	Sync.SyncScheduler	DEBUG	Woke from sleep.
1648351922563	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1648351922563	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1648351922563	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648351924563	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648351924563	Sync.Service	INFO	Starting sync at 2022-03-27 09:02:04 in browser session QYCP8427cDk4
1648351924564	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1648351924564	Sync.Service	INFO	Cannot start sync: already syncing?
1648351929022	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1648351929022	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1648351929024	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1648351929025	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648351929026	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648351929026	Sync.Service	INFO	Starting sync at 2022-03-27 09:02:09 in browser session QYCP8427cDk4
1648351929026	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1648351929026	Sync.Service	INFO	Cannot start sync: already syncing?
1648351930487	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648351931522	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections failed: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1648351931523	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1648351931523	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1648351931526	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
